### PR TITLE
RDR+CNV app deployment and failover/relocate

### PR DESCRIPTION
**Pull request by sidhant-agrawal**
Added tests to verify CNV based application (Subscription and ApplicationSet) deployment and failover/relocate between managed clusters on a RDR setup

CNV workload added in ocs-workloads: 
 - https://github.com/red-hat-storage/ocs-workloads/pull/66
 - https://github.com/red-hat-storage/ocs-workloads/pull/73

### CODE REVIEW

File: tests/functional/disaster-recovery/regional-dr/test_cnv_app_failover_and_relocate.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+from ocs_ci.helpers import dr_helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import wait_for_nodes_status, get_node_objs
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.utility.utils import ceph_health_check
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
```
Comment (am-agrawa) [inline tests/functional/disaster-recovery/regional-dr/test_cnv_app_failover_and_relocate.py:17]: Should we keep it to tier1?

File: tests/functional/disaster-recovery/regional-dr/test_cnv_app_failover_and_relocate.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    )
+    def test_cnv_app_failover_and_relocate(
+        self,
+        primary_cluster_down,
+        cnv_dr_workload,
+        nodes_multicluster,
+        node_restart_teardown,
+    ):
+        """
+        Tests to verify CNV based application (Subscription and ApplicationSet) deployment and
```
Comment (am-agrawa) [inline tests/functional/disaster-recovery/regional-dr/test_cnv_app_failover_and_relocate.py:45]: Can we pls add a doc. text that it is only supported for RBD volumes?

File: tests/functional/disaster-recovery/regional-dr/test_cnv_app_failover_and_relocate.py (hunk 1 above)
Comment (am-agrawa) [reply tests/functional/disaster-recovery/regional-dr/test_cnv_app_failover_and_relocate.py:17]: We prefer to have it as tier1, discussed in DR meeting

**Comment by prsurve** [comment]
@sidhant-agrawal is it possible to move md5sum validation to utils or helper could be helpful for discoveredapps

**Review by prsurve — APPROVED** [review]

**Review by keesturam — APPROVED** [review]
clean implementation, great work @sidhant-agrawal

**Pull request merged**